tests/std/tests/P0019R8_atomic_ref/test.cpp

```diff
@@ -269,6 +269,26 @@ void test_ptr_ops() {
     assert(ry.load() == a + 0x10);
 }
 
+// GH-1497 <atomic>: atomic_ref<const T> fails to compile
+void test_gh_1497() {

```

Comment:
`test_gh_1497` isn't very descriptive. Maybe rename it to `test_const` or `test_const_value_type`?

Reply:
Sure, how about `test_gh_1497_const_type`? This mentions both the bug number and what's being tested (the former is fairly common throughout our tests; it's a reminder that the function is testing something that was specifically broken, instead of being a general test for correctness). I thought that `const_value_type` might be confusing if the Standard ever changes its definition of `atomic_ref::value_type` to `remove_cv_t`, but `const` might sound like it was describing top-level constness on the `atomic_ref` itself, so `const_type` seemed reasonably descriptive.